tests/features/search/search.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const bookETLDataEntity = 'Book_ETL_aqa';
+const specialCharactersDataEntity = '_AirFlights';
+const numbersDataEntity = '737boeing';
+
+test.describe('Search', () => {
+  test.beforeEach(async ({ steps: { pages }, page }) => {
+    await test.step(`I open Catalog page`, async () => {
+      await page.goto('');
+      await pages.topPanel.clickTab('Catalog');
+    });
+  });
+  test.describe('Search by single word', () => {
+    /**
+     * /project/1/test-cases/17
+     */
+    test(`Should contain expected item with single word valid expression`, async ({
+      steps: { pages },
+    }) => {
+      await test.step(`Fill valid expression in search input`, async () => {
+        await pages.catalog.fillSearchBar(bookETLDataEntity);
+        await pages.catalog.confirmSearch();
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/20
+     */
+    test(`Should display empty list with single word invalid expression`, async ({
+      steps: { pages },
+    }) => {
+      await test.step(`Fill invalid expression in search input`, async () => {
+        await pages.catalog.fillSearchBar('bookETLDataEntity');
+        await pages.catalog.confirmSearch();
+        expect(await pages.catalog.isListEmpty()).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/21
+     */
+    test(`Should contain expected item with single word expression which starts with special characters`, async ({
+      steps: { pages },
+    }) => {
+      await test.step(`Fill single word expression which starts with special characters in the input`, async () => {
+        await pages.catalog.fillSearchBar(specialCharactersDataEntity);
+        await pages.catalog.confirmSearch();
+        expect(await pages.catalog.isListItemVisible(specialCharactersDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/35
+     */
+    test(`Should contain expected item with single word expression which starts with numbers`, async ({
+      steps: { pages },
+    }) => {
+      await test.step(`Fill single word expression which starts with numbers in the input`, async () => {
+        await pages.catalog.fillSearchBar(numbersDataEntity);
+        await pages.catalog.confirmSearch();
+        expect(await pages.catalog.isListItemVisible(numbersDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/19
+     */
+    test(`Should display full list when searching by empty string`, async ({
+      steps: { pages },
+    }) => {
+      await test.step(`Empty search string`, async () => {
+        await pages.catalog.clickSearchButton.click();
+        expect(await pages.catalog.amountInTab('All', 474)).toBeTruthy();
```

Comment:
Will this test fail in case we add new entities for our test scenarios?

Reply:
I think so, we should update it if we add new data-entities

Reply:
@damirabdul, @makonakro I think the following assert will be enough for this test.

```
expect(await pages.catalog.isListEmpty()).toBeFalsey();
```

Any ideas?